APIManager.js

```diff
@@ -0,0 +1,83 @@
+class APIManager {
+    constructor() {
+        this.data = {}
+    }
+    loadUser() {
```

Comment:
This is the format you should follow. It is also in the kernel
loadUser= () => {
$.ajax({
method: 'GET',
url: 'https://randomuser.me/api/',
dataType: 'json',
success: (response) => {
this.data["firstName"] = response.results[0].name.first;
this.data["lastName"] = response.results[0].name.last;
this.data["city"] = response.results[0].location.city;
}
},
error: (xhr, text, error) => {
console.log(text)
},
})
}